scipy/stats/tests/test_multivariate.py

```diff
@@ -2574,7 +2574,7 @@ class TestMultivariateHypergeom:
             # test for `n < 0`
             ([3, 4], [5, 10], -7, np.nan),
             # test for `x.sum() != n`
-            ([3, 3], [5, 10], 7, np.NINF)
+            ([3, 3], [5, 10], 7, inp.inf)
```

Comment:
```suggestion
            ([3, 3], [5, 10], 7, -np.inf)
```

Reply:
argh, forgot to save, sorry for the noise there